pkg/models/paths/paths_scenes.go

```diff
@@ -49,6 +49,10 @@ func (sp *scenePaths) GetSpriteImageFilePath(checksum string) string {
 	return filepath.Join(sp.generated.Vtt, checksum+"_sprite.jpg")
 }
 
+func (sp *scenePaths) GetCaptionPath(checksum string) string {
+	return filepath.Join(sp.generated.Vtt, checksum+".vtt")
+}
+
```

Comment:
Is this function used anywhere? Aren't vtts loaded directly ?

Reply:
It's not used. I'll remove it.